tests/calculate.sh

```diff
@@ -6,5 +6,5 @@ set -o pipefail
 #enter the workflow's final output directory ($1)
 cd $1
 
-cat *.crosscheck_metrics.txt | grep -vE "^#" | md5sum
-cat *.crosscheck_metrics.matrix | md5sum
+cat *.crosscheck_metrics.txt | grep -vE "^#"
```

Comment:
curious as to why you updated from md5sums to string comparison? 
easier troubleshooting when build fails?

Reply:
Correct. When md5sum fails, it tells me nothing about the differences that caused the failure. I have to manually compare the files. This ensures when there are differences, they are reported in the Jankins log (comparison is automatic).